data/126/021/452/3/1260214523.geojson

```diff
@@ -28,18 +28,24 @@
     "lbl:max_zoom":18.0,
     "lbl:min_zoom":13.5,
     "mz:hierarchy_label":1,
-    "mz:is_current":-1,
+    "mz:is_current":1,
     "mz:min_zoom":12.0,
+    "name:eng_x_preferred":[
+        "Tomba Kanssa"
+    ],    "name:eng_x_variant":[
```

Comment:
Hmm.. looks like invalid geojson? I also see an extra return char at the tail end of this record. If you're able to Exportify the record, this should correct itself..